docs/extending.md

```diff
@@ -1,195 +1,36 @@
-# Extending Netlify CMS
-The Netlify CMS exposes an `window.CMS` global object that you can use to register custom widgets, previews and editor plugins. The available methods are:
+## Extending With Widgets
+
+The NetlifyCMS exposes an `window.CMS` global object that you can use to register custom widgets, previews and editor plugins. The available widger extension methods are:
 
-* **registerPreviewStyle** Register a custom stylesheet to use on the preview pane.
-* **registerPreviewTemplate** Registers a template for a collection.
 * **registerWidget** lets you register a custom widget.
 * **registerEditorComponent** lets you add a block component to the Markdown editor
 
-**Writing React Components inline**
```

Comment:
This content was moved to the customization section: https://www.netlifycms.org/docs/customization/

We had a discussion prior to launch and came to the decision that extending the cms with widgets (this section) and customization of the preview pane (new section) are two separate things and now live in two seprate sections.